src/map/clif.cpp

```diff
@@ -14757,6 +14757,28 @@ void clif_parse_GM_Item_Monster(int fd, map_session_data *sd)
 	}
 }
 
+/// /resetcooltime 
+/// 0a88 (CZ_CMD_RESETCOOLTIME).
+void clif_parse_GMCooldownReset(int fd, map_session_data* sd) {
+#if PACKETVER >= 20160622
+	if(sd->group_id < 99)
+		return;
+	for (int i = 0; i < MAX_SKILLCOOLDOWN; i++) {
+		if( sd->scd[i] != nullptr ) {
+			std::string buf = "Found skill '" + std::string(skill_db.find(sd->scd[i]->skill_id)->name) + "', unblocking...";
+			clif_displaymessage(sd->fd, buf.c_str());
+			buf.clear();
```

Comment:
```suggestion
			std::string buf = "Found skill '" + std::string(skill_db.find(sd->scd[i]->skill_id)->name) + "', unblocking...";
			clif_displaymessage(sd->fd, buf.c_str());
			buf.clear();
```
Did you add this for debugging or how do you know that this message is being triggered on AEGIS?

Reply:
> Did you add this for debugging or how do you know that this message is being triggered on AEGIS?

I was looking for how to use the 0x01df package in-game, I think it was removed, then I saw some commands in the assembly that are giving unknowpacket  and disconnect myself, one of them was this, the message i got from hercule